### PR TITLE
Identity Oracle

### PR DESCRIPTION
**Pull request by iamsahu**
An identity oracle which returns an exchange rate of 1:1 taking decimals into consideration for the conversion

### CODE REVIEW

**Comment by ultrasecreth** [comment]
If it's making any modification, then it's not identity anymore 😉

**Review by alcueca — APPROVED** [review]
Thanks!

**Comment by alcueca** [comment]
@ultrasecreth, calling it `IdentityMeaningPriceOfOneButAccountingForDecimals.sol` sounds a bit too wordy, I'm happy with the current name :P

**Comment by ultrasecreth** [comment]
> @ultrasecreth, calling it `IdentityMeaningPriceOfOneButAccountingForDecimalsMultiOracle.sol` sounds a bit too wordy, I'm happy with the current name :P

Identity refers to the math property of a function that returns exactly the same as it's input, which this code no longer do.

This is more like a "pegged" oracle or whatever, but it's def not an identity anymore 🤓
